src/components/header/PrimaryNav/PrimaryNav.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { NavCloseButton } from '../NavCloseButton/NavCloseButton'
+import { NavList } from '../NavList/NavList'
+
+type PrimaryNavProps = {
+  items: React.ReactNode[]
+  onToggleMobileNav:
```

Comment:
same comment as above, you can do `onToggleMobileNav?` instead of including `| undefined` on the type